recipes/hwloc/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
You can drop ``test_v1_package`` all together, it is no longer required for ConanCenter.